Dockerfile

```diff
@@ -10,15 +10,28 @@ WORKDIR /usr/src/app
 COPY package.json ./
 COPY yarn.lock ./
 # RUN node --stack-size=6000 $(which npm) --verbose install
-RUN yarn install
+RUN yarn install 
 
 # copy all the stuff
 COPY . .
 
 # generate the prisma client, then run the build which will generate the static site and all other assets
-RUN npx prisma generate && yarn build
+RUN npx prisma generate && yarn add --dev typescript @types/node && yarn build
+# prune reduces the image some bits more :)
+RUN npm prune --production
+
+
+## Shrink final image, copy built nextjs and startup the server
+FROM node:12-alpine
+
+WORKDIR /usr/src/app
+
+# copy from build image
+COPY --from=build /usr/src/app/package.json ./package.json
```

Comment:
This and the next line are not needed. Also would it be clearer to add all the stuff we **do not want** to the .gitignore file? 

Also since the WORKDIR was set on line 27, is it necessary to specify the full path?
@jorgerasillo

Reply:
1. I prefer to work with full paths vs. relative paths, having said I can change the destination to be full paths
2. for setting the WORKSPACE dir, same as 1
3. re: .gitignore, don't think so, since all the files are being generated inside the container so nothing is generated locally and therefore no reason to add them to the .gitignore file